test/mason/publish-tests/publishHelp.good

```diff
@@ -8,5 +8,6 @@ Options:
     -h, --help                   Display this message
     --dry-run                    Check to see if package is ready to be published
     --no-update                  Prevents registrys from being updated when a package is published.
+    --update                     Override MASON_OFFLINE=true and enable internet access.
```

Comment:
What if a user has a completely locally hosted registry, but no internet access? I assume they would want to update in that mode still. Is it possible for us to check for that before disabling the update with `MASON_OFFLINE=true`?